db/migrate/20230825233438_remove_author_id_from_posts.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAuthorIdFromPosts < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :posts, :author_id, :integer
+  end
+end
```

Comment:
- [ ] also this is causing an issue in running your migration file, kindly remove it too